packages/Liquid-Tests.package/LQLoadTest.class/instance/testTwoHundredUsers.st

```diff
@@ -13,8 +13,10 @@ testTwoHundredUsers
 			 yourself).
 			
 	pollId := 'gotta-catch-rayquaza'.
+	password := 'karpador'.
```

Comment:
[Boulder] You manually set a Password here, circumventing `LQPasswordManager>>generatePasswordFor:`, which adds the password to its store. This also suggests to readers of the test that you can just choose your own passwords, which (for the before reason) is something we _explicitly_ do not support, since the code would break when using `closePoll`, the actual way polls should be closed - which we of course need to skip in the test because of the User Input required.
If you leave out choosing your own password, the test also gets significantly shorter - you don't need the `password` temporary variable, can use `LiquidPoll>>startWithId:ifTaken:` instead of `LiquidPoll>>startWithId:andPassword:ifTaken:`.
Edit: Remember to remove the automatically generated password entry from the store after the test is finished to not fill it up unnecessarily.